sanic_jwt/initialization.py

```diff
@@ -0,0 +1,174 @@
+from sanic import Blueprint
+from sanic import Sanic
+from sanic.views import HTTPMethodView
+from sanic_jwt import exceptions
+from sanic_jwt import endpoints
+from sanic_jwt.authentication import Authentication
+from sanic_jwt.configuration import Configuration
+from sanic_jwt.response import Response
+from sanic_jwt.response import make_response
+
+
+def initialize(*args, **kwargs):
```

Comment:
Shouldn't we add some checks to see if `args[0]` is an instance of Sanic instead of a bogus and etc?

Reply:
Hmm, I'm reading the code further I realize (correct me if I'm wrong) that we can apply it to a `bp` as well? Is this functionality working?

Reply:
There is a check and it raises an exception if the instance is not a sanic appear blueprint. Also I have some tests in place for this.but as mentioned by you in #63, still more is needed.

Reply:
Allright! I think we need to create some more issues ...